Description: Add po/Makevars and po/LINGUAS for the modern gettext infrastructure
 Declaring AM_GNU_GETTEXT_REQUIRE_VERSION makes autopoint replace the 1998-era
 po/Makefile.in.in with the one from the host's gettext (>= 0.19). That modern
 Makefile.in.in has a build rule depending on po/Makevars, which this package
 never shipped, so make aborts with "No rule to make target 'Makevars'".
 .
 po/Makevars carries the canonical gettext variables (message domain, xgettext
 keywords, copyright holder, ...). po/LINGUAS lists the translations to build;
 without it the modern infrastructure reads an empty language set and silently
 drops the existing German catalogue (de.po), so it is added with "de".
Author: Martin A. Godisch <godisch@debian.org>
Last-Update: 2026-06-04
Forwarded: not-needed
---
--- /dev/null
+++ lrzsz-0.12.21rc/po/Makevars
@@ -0,0 +1,36 @@
+# Makefile variables for PO directory in any package using GNU gettext.
+
+# Usually the message domain is the same as the package name.
+DOMAIN = $(PACKAGE)
+
+# These two variables depend on the location of this directory.
+subdir = po
+top_builddir = ..
+
+# These options get passed to xgettext.
+XGETTEXT_OPTIONS = --keyword=_ --keyword=N_
+
+# This is the copyright holder that gets inserted into the header of the
+# $(DOMAIN).pot file.  Set this to the copyright holder of the surrounding
+# package.  (Note that the msgstr strings, extracted from the package's
+# sources, belong to the copyright holder of the package.)  Translators are
+# expected to transfer the copyright for their translations to this person
+# or entity, or to disclaim their copyright.  The empty string stands for
+# the public domain; in this case the translators are expected to disclaim
+# their copyright.
+COPYRIGHT_HOLDER = Free Software Foundation, Inc.
+
+# This tells whether or not to prepend "GNU " prefix to the package
+# name that gets inserted into the header of the $(DOMAIN).pot file.
+# Possible values are "yes", "no", or empty.  If it is empty, try to
+# detect it automatically by scanning the files in $(top_srcdir) for
+# "GNU packagename" string.
+PACKAGE_GNU =
+
+# This is the email address or URL to which the translators shall report
+# bugs in the untranslated strings.  It can be the empty string.
+MSGID_BUGS_ADDRESS =
+
+# This is the list of locale categories, beyond LC_MESSAGES, for which the
+# message catalogs shall be used.  It is usually empty.
+EXTRA_LOCALE_CATEGORIES =
--- /dev/null
+++ lrzsz-0.12.21rc/po/LINGUAS
@@ -0,0 +1,2 @@
+# Set of available languages.
+de
